python/shotgun_desktop/errors.py

```diff
@@ -135,26 +135,27 @@ def __init__(self, reason, toolkit_path):
         )
 
 
-class ToolkitDisabledError(ShotgunDesktopError):
```

Comment:
The code raising this exception was removed a while back, but we forgot to remove the exception itself.